test/test_transforms_tensor.py

```diff
@@ -762,12 +763,19 @@ def shear(pil_img, level, mode, resample):
     )
     assert out == expected_out
 
-    if interpolation in (F.InterpolationMode.NEAREST, ):
-        # Check tensor output vs expected pil
-        out = _apply_op(
-            t_img, op_name=f"Shear{mode}", magnitude=arc_level, interpolation=interpolation, fill=0
-        )
-        _assert_approx_equal_tensor_to_pil(out, expected_out)
+    if interpolation == F.InterpolationMode.BILINEAR:
+        # We skip bilinear mode for tensors as
+        # affine transformation results are not exactly the same
+        # between tensors and pil images
+        # MAE as around 1.40
+        # Max Abs error can be 163 or 170
+        return
```

Comment:
Shall we remove it from the `parametrize` then?

Reply:
We can't remove it from parametrize as still checking that for torchvision out PIL vs expected PIL, just above, L758 (`assert out == expected_out`)